ffi/tests/ctests/src/tests.c

```diff
@@ -121,6 +121,53 @@ void wirefilter_ffi_ctest_parse_bad_filter() {
     wirefilter_free_scheme(scheme);
 }
 
+void wirefilter_ffi_ctest_filter_uses_field() {
+    wirefilter_scheme_t *scheme = wirefilter_create_scheme();
+    rust_assert(scheme != NULL, "could not create scheme");
+
+    wirefilter_add_type_field_to_scheme(
```

Comment:
Maybe remove unnecessary fields to reduce noise?

Reply:
Why specifically for this test?

Reply:
Because I haven't noticed this on others tests 🤷🏻‍♂️

Alternatively, move out scheme initialisation into a helper function?

Reply:
I would prefer that :+1: